extensions/resource-deployment/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "resource-deployment",
+  "displayName": "Azure Data Resource Deployment",
+  "description": "Azure Data Resource Deployment",
+  "version": "0.0.1",
+  "publisher": "Microsoft",
+  "preview": true,
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
+  "icon": "images/sqlserver.png",
+  "aiKey": "AIF-5574968e-856d-40d2-af67-c89a14e76412",
+  "engines": {
+    "vscode": "*",
+    "azdata": "^1.6.0"
+  },
+  "activationEvents": [
+    "onCommand:azdata.resource.sql-image.deploy",
+    "onCommand:azdata.resource.sql-bdc.deploy"
+  ],
+  "main": "./out/main",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/azuredatastudio.git"
+  },
+  "extensionDependencies": [
+    "Microsoft.mssql"
+  ],
+  "contributes": {
+    "commands": [
+      {
+        "command": "azdata.resource.sql-image.deploy",
+        "title": "%deploy-sql-image-command-name%",
+        "category": "%deploy-resource-command-category%"
+      },
+      {
+        "command": "azdata.resource.sql-bdc.deploy",
+        "title": "%deploy-sql-bdc-command-name%",
+        "category": "%deploy-resource-command-category%"
+      }
+    ],
+    "dataExplorer.actions": [
+      {
+        "commandId": "azdata.resource.sql-image.deploy",
+        "label": "%deploy-sql-image-menu-item-text%",
+        "category": "%deploy-resource-command-category%",
+        "isPrimary": false
+      },
+      {
+        "commandId": "azdata.resource.sql-bdc.deploy",
+        "label": "%deploy-sql-bdc-menu-item-text%",
```

Comment:
Shouldn't this just come from the command definition? I understand UX may want to have different labels depending on entry point, but that doesn't make a huge amount of sense here. If we can align closer with how menu contributions work today, I'd be happier.

Reply:
updated.